docs/usage/model_config.md

```diff
@@ -346,3 +350,40 @@ print(Model(x=[1, '2']))
 print(Model(x=[1, 2]))
 #> x=[1, 2]
 ```
+
+
+## Protected Namespaces
+
+Protected namespaces prevent model from having field that starts with them.
+It can be a list of strings and the default value is `['model_']`.
+
+```py
+from pydantic import BaseModel
+
+try:
+
+    class Model(BaseModel):
+        model_x: str
```

Comment:
IMO, it's easier to follow the logic of examples when field and vars have meaningful names.

Something like `model_prefixed_field` would be better here as well as `protect_me_`/`also_protect_` instead of `p1_`/`p2_` below.